scripts/variantstore/wdl/GvsCreateVAT.wdl

```diff
@@ -604,3 +619,72 @@ task BigQueryExportVat {
+
+        if [ ~{has_service_account_file} = 'true' ]; then
+            gsutil cp ~{service_account_json_path} local.service_account.json
+            gcloud auth activate-service-account --key-file=local.service_account.json
+            gcloud config set project ~{project_id}
+        fi
+
+        mkdir TSVs
+        contigs=( ~{sep=' ' contig_array} )
+        files=" header.gz"
+
+        echo_date "looping over contgs: $contigs"
+        for i in "${contigs[@]}"
+        do
+            echo_date "copying files from ~{output_path}export/$i/*.tsv.gz"
+            gsutil -m cp ~{output_path}export/$i/*.tsv.gz TSVs/
+            echo_date "concatenating local tsv.gz files"
+            cat TSVs/*.tsv.gz > vat_$i.tsv.gz
+            echo_date "removing now concatenated files"
+            rm TSVs/*.tsv.gz
+            files="$files vat_$i.tsv.gz"
+        done
+
+        echo_date "making header.gz"
+        echo "vid transcript contig position ref_allele alt_allele gvs_all_ac gvs_all_an gvs_all_af gvs_all_sc gvs_max_af gvs_max_ac gvs_max_an gvs_max_sc gvs_max_subpop gvs_afr_ac gvs_afr_an gvs_afr_af gvs_afr_sc gvs_amr_ac gvs_amr_an gvs_amr_af gvs_amr_sc gvs_eas_ac gvs_eas_an gvs_eas_af gvs_eas_sc gvs_eur_ac gvs_eur_an gvs_eur_af gvs_eur_sc gvs_mid_ac gvs_mid_an gvs_mid_af gvs_mid_sc gvs_oth_ac gvs_oth_an gvs_oth_af gvs_oth_sc gvs_sas_ac gvs_sas_an gvs_sas_af gvs_sas_sc gene_symbol transcript_source aa_change consequence dna_change_in_transcript variant_type exon_number intron_number genomic_location dbsnp_rsid gene_id gene_omim_id is_canonical_transcript gnomad_all_af gnomad_all_ac gnomad_all_an gnomad_failed_filter gnomad_max_af gnomad_max_ac gnomad_max_an gnomad_max_subpop gnomad_afr_ac gnomad_afr_an gnomad_afr_af gnomad_amr_ac gnomad_amr_an gnomad_amr_af gnomad_asj_ac gnomad_asj_an gnomad_asj_af gnomad_eas_ac gnomad_eas_an gnomad_eas_af gnomad_fin_ac gnomad_fin_an gnomad_fin_af gnomad_nfr_ac gnomad_nfr_an gnomad_nfr_af gnomad_sas_ac gnomad_sas_an gnomad_sas_af gnomad_oth_ac gnomad_oth_an gnomad_oth_af revel splice_ai_acceptor_gain_score splice_ai_acceptor_gain_distance splice_ai_acceptor_loss_score splice_ai_acceptor_loss_distance splice_ai_donor_gain_score splice_ai_donor_gain_distance splice_ai_donor_loss_score splice_ai_donor_loss_distance omim_phenotypes_id omim_phenotypes_name clinvar_classification clinvar_last_updated clinvar_phenotype" | gzip > header.gz
+
+        echo_date "concatenating $files"
+        cat $(echo $files) > vat_complete.tsv.gz
+        echo_date "bgzipping concatenated file"
+        cat vat_complete.tsv.gz | gunzip | bgzip > vat_complete.bgz.tsv.gz
+        echo_date "copying bgzipped file to ~{output_path}"
+        gsutil -m cp vat_complete.bgz.tsv.gz ~{output_path}
+    >>>
+    # ------------------------------------------------
+    # Runtime settings:
+    runtime {
+        docker: "us.gcr.io/broad-dsde-methods/variantstore:ah_var_store_2022_05_03"
+        memory: "2 GB"
```

Comment:
this is really small, with all the `gsutil -m cp`ing a bigger machine with a bigger disk would probably make this run a lot faster

Reply:
There is a parameter to overwrite the disk size.  What do you think a good default is (right now it's 50)?

Reply:
I'm not sure how much you're copying down here but I generally don't hesitate to start at 500. I went [a little bigger](https://github.com/broadinstitute/gatk/blob/0c9ef521df06433aa2aabdb05c01353bde26e8d9/scripts/variantstore/wdl/GvsIngestTieout.wdl#L147) on my VS-261 tieout task.

Reply:
For a 15K sample callset the individual `tsv.gz` files added up to 9.3 GiB; for Charlie, 52.48 GiB.

Reply:
going with a default of 250